src/Statistics/Service/StatisticsService.php

```diff
@@ -119,6 +141,25 @@ public function getUniqueParticipantCountsForPartnersByYear(Year $year): ?array
         return $counts;
     }
 
+    /**
+     * @param UuidInterface $companyId
+     * @return EmployeeParticipationShare
+     * @throws InvalidArgumentException
+     */
+    public function getEmployeeParticipationShare(UuidInterface $companyId): EmployeeParticipationShare
```

Comment:
Het is al zeer goed dat deze methode in service zit, maar misschien moeten we een nieuwe `DashboardService` maken. Dan is een betere `separation of concerns`.